src/test/java/hw3/ex2/ServiceMenusAndDifferentElementsPageTest.java

```diff
@@ -0,0 +1,82 @@
+package hw3.ex2;
+
+import hw3.AbstractBaseSeleniumTest;
+import hw3.elementsEnum.MenuElements;
+import hw3.elementsEnum.MenuType;
+import hw3.elementsEnum.ServiceMenuElements;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class ServiceMenusAndDifferentElementsPageTest extends AbstractBaseSeleniumTest {
+    private ServicePageSteps steps;
+
+    @BeforeMethod
+    @Override
+    public void setUp() {
+        super.setUp();
+        steps = new ServicePageSteps(driver);
+    }
+
+    @Parameters({"url", "pageTitle", "login", "password", "name",
+            "checkBoxWater", "checkBoxWind",
+            "radioAttribute", "radioValue",
+            "dropDownElement", "dropDownValue"})
+
+    @Test
+    public void serviceMenusAndDifferentElementsPageTest(String url, String pageTitle, String login, String password, String name,
+                                                         String checkBoxWater, String checkBoxWind,
+                                                         String radioAttribute, String radioValue,
+                                                         String dropDownElement, String dropDownValue) {
+
```

Comment:
Why do you need such amount of the parameters

Reply:
Because there was a requirement to not hard code test data, so I decided to export almost all checked parameters.